SROS2_Linux.md

```diff
@@ -141,29 +138,17 @@ scp -r talker USERNAME@oldschool.local:~/sros2_demo/demo_keys
 That will be very quick, since it's just copying some very small text files.
 Now, we're ready to run a multi-machine talker/listener demo!
 
-Once the envrionment setup we can run on oldschool, for either Fast-RTPS:
+Once the envrionment setup we can run on oldschool:
```

Comment:
Nit:

`Once the envrionment setup` -> `Once the environment is setup`

Reply:
thanks ! please push typo/nitpicks directly to the branch if possible